src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ThemeMode.cs

```diff
@@ -14,29 +18,113 @@
 
 namespace System.Windows
 {
+    /// <summary>
+    /// ThemeMode structure describes the Fluent theme mode to be applied
+    /// on Application or a Window.
+    /// </summary>
+    /// <remarks>
+    /// This is an experimental API and may be modified or removed in future releases.
+    /// Since this is an experimental API, rather than creating a new instance of ThemeMode,
+    /// use the static properties Light, Dark, System and None.
+    /// </remarks>
     [Experimental("WPF0001")]
     public readonly struct ThemeMode : IEquatable<ThemeMode>
     {
+        /// <summary>
+        /// Predefined theme mode : None.
+        /// This is the default value for Application.ThemeMode and Window.ThemeMode
+        /// and it means that Fluent theme will not be applied on the Application or Window.
+        /// </summary>
+        /// <remarks>
+        /// In case, when Application.ThemeNode is not set to None,
+        /// even if Window.ThemeMode is set to None, the Fluent theme will be applied on the Window.
+        /// </remarks>
         public static ThemeMode None => new ThemeMode();
+        
+        /// <summary>
+        /// Predefined theme mode : Light.
+        /// Whenever this mode is set on Application or Window, 
+        /// Fluent Light theme will be applied.
+        /// </summary>
         public static ThemeMode Light => new ThemeMode("Light");
+        
+        /// <summary>
+        /// Predefined theme mode : Dark.
```

Comment:
Nit: Is there a reason for the whitespace before colon? Doesn't feel natural.

Reply:
Just one of my habits. Will update this

Reply:
No worries, it is really a nit that I was considering not posting but a standard in english writing as well, so I did.